app/sidekiq/decision_review/delete_saved_claim_records_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module DecisionReview
+  class DeleteSavedClaimRecordsJob
+    include Sidekiq::Job
+
+    # No need to retry since the schedule will run this periodically
+    sidekiq_options retry: false
+
+    def perform
+      return unless enabled?
+
+      ::SavedClaim.where(delete_date: ..DateTime.now).destroy_all
```

Comment:
Do you anticipate the possibility of errors? should there be a rescue and a Rails.logger.error?

Reply:
I'll add a rescue and log the error for any exceptions.